src/streams/receive.rs

```diff
@@ -597,6 +615,23 @@ impl StreamRx {
         self.rtx = Some(rtx);
         self.register_rtx = None;
     }
+
+    /// Reset the current rollover counter (ROC).
+    ///
+    /// This is used in scenarios where we use a single sequence number across all
+    /// receivers of the same stream (as opposed to a sequence number unique per peer).
+    ///
+    /// > Receivers joining an on-going session MUST be given the
+    /// > current ROC value using out-of-band signaling such as key-management
+    /// > signaling.  Furthermore, the receiver SHALL initialize s_l to the RTP
+    /// > sequence number (SEQ) of the first observed SRTP packet (unless the
+    /// > initial value is provided by out of band signaling such as key
+    /// > management).
+    pub fn reset_roc(&mut self, roc: u64) {
```

Comment:
Don't we need to be able to set the RTX ROC too? If a session has been going for long enough the RTX ROC might also be non-zero when the receiver joins

Reply:
The RTX is peer-peer (not entire session), and it's not possible for the user to chose a value for it.

Reply:
Cool 👍🏼